inc/custom-logo.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Sample implementation of the Custom Logo feature
+ *
+ * <?php _s_the_custom_logo(); ?>
+ *
+ * @package _s
+ */
+
+/**
+ * Get the custom Logo attachment link
+ *
+ */
+function _s_the_custom_logo() {
+	if ( function_exists( '_s_the_custom_logo' ) ) {
```

Comment:
Calling the `function_exists()` from within the function it checks for, however, will always return `true`.

I think you may have meant to wrap the whole function declaration within a `if ( ! function_exists( '_s_the_custom_logo' ) ) {}` wrapper ?